### PR TITLE
roll r2r jobs into crossgen2-outerloop

### PR DESCRIPTION
**Pull request by mangod9**
With crossgen1 now defunct merging R2R and crossgen2 outerloops runs into a single CI. This should help with reducing duplication of CI runs. 

Contributes to https://github.com/dotnet/runtime/issues/58120

### CODE REVIEW

**Review by trylek — APPROVED** [review]
LGTM, thank you!

**Review by BruceForstall — CHANGES_REQUESTED** [review]
You need to remove the `# Limited outerloop testing in non-composite mode` comment, right?

Are you going to adjust the schedule to be daily, like r2r.yml?

There is still cg2 R2R testing in "outerloop" (ci.yml), that is batch run. Should you remove any testing here that is already done in ci.yml? (Is there any overlap?) Presumably, we do want that batch test R2R testing preferably to this daily "cg2 outerloop" testing.

I'm not sure we need to test on Windows arm(32). We do minimal testing in outerloop to keep it alive, but it is not supported. Maybe it's still useful to keep it alive for R2R?

It might be possible/easier to use `platformGroup: all` instead of listing all the platforms; I'm not sure if your platform set is equivalent to that (but note that `platformGroup: all` includes windows_arm).

**Comment by mangod9** [comment]
yeah makes sense to remove arm, though I copied the it over from r2r. Will get that removed. Will check ci.yml as to what is run there. Some duplication is ok unless its easy to separate.

File: eng/pipelines/coreclr/crossgen2-outerloop.yml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 trigger: none
 
 schedules:
-- cron: "0 7 * * 0,2,4"
+- cron: "0 5 * * *"
   displayName: Sun, Tue, Thu at 11:00 PM (UTC-8:00)
```
Comment (BruceForstall) [inline eng/pipelines/coreclr/crossgen2-outerloop.yml:5]: Update this comment

Comment (mangod9) [reply eng/pipelines/coreclr/crossgen2-outerloop.yml]: yup thanks for noticing.

**Review by BruceForstall — APPROVED** [review]

File: eng/pipelines/coreclr/crossgen2-outerloop.yml [hunk 2]
```diff
@@ -15,12 +15,15 @@ jobs:
...
     - Linux_arm64
     - OSX_arm64
     - OSX_x64
+    - windows_x86    
     - windows_x64
     - windows_arm64
+    - windows_arm
     - CoreClrTestBuildHost # Either OSX_x64 or Linux_x64
     jobParameters:
       testGroup: outerloop
```
[the remark this answers is not in the document]
Comment (mangod9) [reply eng/pipelines/coreclr/crossgen2-outerloop.yml]: will update in a subsequent PR. @davidwrighton any specific reason comparison jobs require Release builds?

**Pull request merged**